### ISSUE TITLE
Equation rendering with \ifelse{html}{\out{}}{\eqn{}} in Rd

### ISSUE THREAD

**Issue by aravind-j** [r-lib/pkgdown#556]
In `Rd` files, equations rendered with `\eqn{}` or `\deqn{}` work fine. However, if the syntax is of the format, `\ifelse{html}{\out{}}{\eqn{}}`, the output is not rendered.

for example

`\ifelse{html}{\out{<p style="text-align: center;"><em>s =
<sup>&part;y</sup>&frasl;<sub>&part;x</sub> =
<sup>abc<sup>b</sup>x<sup>b-1</sup></sup>&frasl;<sub>&radic;[(c<sup>b</sup> +
x<sup>b</sup>)<sup>2</sup>]</sub></em></p>}}{\deqn{s = \frac{\partial
y}{\partial x} = \frac{abc^{b}x^{b-1}}{(c^{b}+x^{b})^{2}}}}`

This syntax is used to get the equation in both HTML help as well as in the PDF help. 
(https://stackoverflow.com/questions/3782337/mathematical-notion-superscripts-in-rd-files, https://stackoverflow.com/questions/22918797/subscript-of-math-equation-in-r-documentation)

**Comment by hadley** [comment]
As far as I can both `\ifelse` and `\out` work:

```R
rd2html("\\ifelse{html}{\\out{a}}{b}")
#> [1] "a"
```

Can you please provide a link to the source .Rd and the generated pkgdown html?

But do note that pkgdown already renders `\eqn{}` with mathjax..

**Comment by aravind-j** [comment]
Updating `pkgdown` has resolved this issue.

```
pkgdown:::rd2html("\\ifelse{html}{\\out{a}}{b}")
[1] "a"
```

However, is it possible to  force output to latex only when rendering html using pkgdown to take advantage of mathjax?

**Comment by hadley** [comment]
I don't see an obvious way. I think you should just use unqualified `\eqn` and expect the Rd rendering engine to do the right thing (which it won't now, but might eventually)
